app/models/authorization.rb

```diff
@@ -39,6 +39,10 @@ def self.omniauth_image(auth)
     end
   end
 
+  def self.external_providers
+    return self.where('provider.ne': 'tomatoes')
```

Comment:
Things I'd do differently:

1. redundant `return`
1. redundant `self`

I like that you used the new hash syntax even with a string key!